subsys/net/lib/nrf_cloud/src/nrf_cloud_codec_internal.c

```diff
@@ -1142,27 +1142,6 @@ int nrf_cloud_get_single_cell_modem_info(struct lte_lc_cell *const cell_inf)
 	return 0;
 }
 
-int nrf_cloud_format_single_cell_pos_req_json(cJSON *const req_obj_out)
-{
-	int err = 0;
-	cJSON *lte_array = cJSON_AddArrayToObjectCS(req_obj_out, NRF_CLOUD_CELL_POS_JSON_KEY_LTE);
-	cJSON *lte_obj = cJSON_CreateObject();
-
-	if (!cJSON_AddItemToArray(lte_array, lte_obj)) {
-		cJSON_Delete(lte_obj);
-		err = -ENOMEM;
-	} else {
-		err = nrf_cloud_json_add_modem_info(lte_obj);
-	}
-
-	if (err) {
-		cJSON_DeleteItemFromObject(req_obj_out, NRF_CLOUD_CELL_POS_JSON_KEY_LTE);
-	}
-
-	return err;
-}
-
-
```

Comment:
Should this also be mentioned in the changelog?

Reply:
sure

Reply:
Really it's a nitpick if I'm being honest